client/src/components/Profile/Profile.js

```diff
@@ -1,8 +1,49 @@
-import React from "react";
+import { React, useEffect, useState } from "react";
```

Comment:
Another way for doing this that I remember seeing more elsewhere is:

```suggestion
import React, { useEffect, useState } from "react";
```